src/test/java/edu/hm/hafner/analysis/parser/JavacParserTest.java

```diff
@@ -274,5 +274,11 @@ private void assertThatWarningIsAtLine(final Issue annotation, final int lineNum
                             "/home/hudson/hudson/data/jobs/Hudson main/workspace/remoting/src/test/java/hudson/remoting/BinarySafeStreamTest.java");
         }
     }
+
+    @Test
```

Comment:
Hey @uhafner 

When I run this test case. I get the following output:

```
java.lang.AssertionError: 
Expecting size of:
  <- (-): 0 issues (0 duplicates)>
to be:
  <1>
but was:
  <0>

```
There is only one `maven-compiler-plugin` in the `tracker_issue63346.log` file. So it should have only one line. 

Does this mean the bug is exposed? 

Where shall I fix it?

Reply:
The bug is not yet exposed. As @KalleOlaviNiemitalo already mentioned you need to replace the 2 place holder texts with an actual warning from [JENKINS-67521](https://issues.jenkins.io/browse/JENKINS-67521) (or https://issues.jenkins.io/browse/JENKINS-63346). (Not that you cannot use the same warnings in both blocks since duplicates are removed). Then your test should pick two actual warnings  (and in JavaDocParserTest) now. Expected count is 1. When that is done, you can fix both parsers, so that the expected will be the actual.

Reply:
After this line:

`[INFO] --- maven-compiler-plugin:3.10.1:testCompile (default-testCompile) @ analysis-model ---`

JavacParser warnings will be recognized?

And after this line:

`[INFO] Building jar: /home/jenkins/workspace/Plugins_analysis-model_master/target/analysis-model-10.23.0-rc4948.f52d79122c41-javadoc.jar`

JavacDoc warning should be recognized?

Am I correct in my understanding?

Reply:
Almost. The compile step can be in different goals, so it might make sense to check for `maven-compiler-plugin` only (without `testCompile`).